### PR TITLE
Updating the GetProfileNameDialog to support localization.

### PR DESCRIPTION
**Pull request by tannergooding**
**Customer scenario**

Users who depend on localization may not be able to create new debug profiles.

**Bugs this fixes:** 

https://devdiv.visualstudio.com/DevDiv/_queries?id=473107&triage=true&_a=edit

**Workarounds, if any**

None.

**Risk**

Low.

**Performance impact**

Low

**Is this a regression from a previous update?**

No

**Root cause analysis:**

This was likely missed when the localization effort was done.

**How was the bug found?**

Customer reported

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @Pilchie, @dotnet/project-system

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml [hunk 1]
```diff
@@ -38,8 +39,8 @@
                 <ColumnDefinition Width="Auto"></ColumnDefinition>
                 <ColumnDefinition Width="Auto"></ColumnDefinition>
             </Grid.ColumnDefinitions>
-            <Button x:Uid="OKButton" Grid.Row="0"  Grid.Column="0" Margin="5" MinWidth="50" Padding="10,2" IsDefault="True" Click="OKButton_Click">OK</Button>
-            <Button x:Uid="CancelButton" Grid.Row="0"  Grid.Column="1" Margin="5" MinWidth="50" Padding="10,2" IsCancel="True">Cancel</Button>
+            <Button x:Uid="OKButton" Grid.Row="0"  Grid.Column="0" Margin="5" MinWidth="50" Padding="10,2" IsDefault="True" Click="OKButton_Click" Content="{x:Static Member=local:PropertyPageResources.OKBtn}" />
```
Comment (Pilchie) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml]: Are there no system standard strings for these?

**Comment by tannergooding** [comment]
FYI. @nguerrera

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml (hunk 1 above)
Comment (tannergooding) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml]: I do not know. Is there some existing VS API for general-use localized strings?

**Comment by Pilchie** [comment]
Roslyn's Extract Interface does roughly the same thing...

https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialog.xaml#L133

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml (hunk 1 above)
Comment (tannergooding) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml]: At least looking through our own code base, we have other resource declarations for OK/Cancel, but nothing that looks to be easily reusable here.

The Roslyn example you gave is in the same boat, it ends up pulling in from a Roslyn resource file declaring the string.

**Comment by nguerrera** [comment]
Looks fine. I'm not aware of a way to get the names from the system.

**Comment by tannergooding** [comment]
@Pilchie is this approved for merge?

**Comment by Pilchie** [comment]
Yes - approved.

**Pull request merged**